packages/img/src/getImage.ts

```diff
@@ -0,0 +1,34 @@
+import { Endpoint, EndpointInstance } from '@rest-hooks/endpoint';
+
+const getImage: EndpointInstance<
+  (this: EndpointInstance, { src }: { src: string }) => Promise<string>,
+  string,
+  undefined
+> = new Endpoint(
+  async function (
+    this: EndpointInstance,
+    { src }: { src: string },
+  ): Promise<string> {
+    return new Promise(resolve => {
+      // even if we polyfill, on server we don't want to actually wait to resolve the Image
+      if (typeof window === 'undefined' || !Image) resolve(src);
```

Comment:
```suggestion
      if (typeof window === 'undefined' || typeof Image !== 'function') resolve(src);
```
without typeof it’s not safe to refer to globales that might not be there.